compiler/src/dotty/tools/dotc/core/tasty/TreeUnpickler.scala

```diff
@@ -951,7 +951,7 @@ class TreeUnpickler(reader: TastyReader,
     }
 
     def isTopLevel(using Context): Boolean =
-      nextByte == IMPORT || nextByte == PACKAGE
+      nextByte == IMPORT || nextByte == EXPORT || nextByte == PACKAGE
```

Comment:
I don' tthink we can have export statements at the top-level in tasty (if they're top-level in a file they'll end up in a top-level object), so this shouldn't be needed.

Reply:
thats right, thanks for spotting